packages/forms-web-app/__tests__/unit/controllers/examination/enter-comment.test.js

```diff
@@ -2,43 +2,179 @@ const {
-const { mockReq, mockRes } = require('../../mocks');
+const { mockReq, mockRes, mockResponse } = require('../../mocks');
+
+const pathShortner = (obj, path) => path.split('.').reduce((value, el) => value && value[el], obj);
+const itemsPath = 'session.examination.selectedDeadlineItems.items';
 
 const pageData = {
 	backLinkUrl: '/examination/select-upload-evidence-or-comment',
-	id: 'examination-enter-comment',
-	pageTitle: 'Enter comment',
-	title: 'Enter comment'
+	id: 'examination-enter-a-comment',
+	pageTitle: 'Make a comment',
+	title: 'Make a comment',
+	hint: 'Any further information requested by',
+	captionTitle: 'Deadline item:',
+	comment: ''
 };
 
-describe('controllers/examination/submitting-for', () => {
+const hintValue = '`${pageData.hint} ${pathShortner(mockRequest, itemsPath)[0].submissionItem}`';
+
+describe('controllers/examination/enter-comment', () => {
 	let req;
 	let res;
 
 	beforeEach(() => {
-		req = mockReq();
 		res = mockRes();
+		req = {
+			...mockReq(),
+			session: {
+				examination: {
+					selectedDeadlineItems: {
+						activeId: '0',
+						items: {
+							0: {
+								complete: false,
+								itemId: '3',
+								submissionItem: 'Statement of Commonality of SoCG ',
+								submissionType: 'comment'
+							}
+						}
+					}
+				}
+			}
+		};
 
 		jest.resetAllMocks();
 	});
 
 	describe('getEnterComment', () => {
-		it('should call the correct template', () => {
+		it('should render the view with default pageData', () => {
+			const mockRequest = { ...req };
+
+			getEnterComment(mockRequest, res);
+
+			expect(res.render).toHaveBeenCalledWith('pages/examination/enter-comment', {
+				...pageData,
+				hint: eval(hintValue)
+			});
+		});
+
+		it('should render the view with default and session pageData', function () {
+			const mockRequest = { ...req };
+
+			const comment = 'I am a comment';
+			mockRequest.session.examination.comment = comment;
+
+			getEnterComment(mockRequest, res);
+
+			expect(res.render).toHaveBeenCalledWith('pages/examination/enter-comment', {
+				...pageData,
+				comment,
+				hint: eval(hintValue)
+			});
+		});
+
+		it('should render not found view', function () {
 			const mockRequest = { ...req };
+			delete mockRequest.session;
+
+			res = mockResponse();
 
 			getEnterComment(mockRequest, res);
 
-			expect(res.render).toHaveBeenCalledWith('pages/examination/enter-comment', pageData);
+			expect(res.render).toHaveBeenCalledWith('error/not-found');
+			expect(res.redirect).not.toHaveBeenCalled();
 		});
 	});
 
-	describe('postSubmittingFor', () => {
-		it('should call the correct template', () => {
+	describe('postEnterComment', () => {
+		it('should redirect user to /examination/comment-has-personal-information-or-not', () => {
+			const mockRequest = { ...req };
+			postEnterComment(mockRequest, res);
+
+			expect(res.render).not.toHaveBeenCalled();
+			expect(pathShortner(mockRequest, itemsPath)[0].submissionType);
```

Comment:
These don't appear to test anything or assert in a way that suggest the test is worth doing. 

Change to `expect(pathShortner(mockRequest, itemsPath));` and the test still passes ?

Reply:
forgot to add the assertion at the end